### DISCUSSION TITLE
q-table get computedRowsNumber

### DISCUSSION THREAD

**Discussion by hlabory** [quasarframework/quasar#14013] [General - Components / Directives / etc] [answered]
Hello:
How can I access the value of computedRowsNumber in a simple way? It is the only way I have been able to determine if the table does not contain values, either because there are none or they are filtered.
I am currently doing this to be able to have a button that with a computed becomes disable/not disable:

```
data () {
  return {
     noHayRegistros : false,
},

computed: {
    getNoHayRegistros () {
      return this.noHayRegistros
},

watch: {
    filtro(){
      this.$nextTick(() => {
        this.noHayRegistros = this.$refs.tabla.computedRowsNumber === 0
      })      
},
```
But I guess there is another easier way.

**Comment by metalsadman** [comment, marked as answer]
yes via ref.
